Widgets/2-FAQs.md

```diff
@@ -53,19 +53,21 @@ Yes, it is possible to make widgets visible on additional pages. The Sezzle widg
 Yes, the widget and modal have built-in translations in French, German, and Spanish for multilingual websites. By default, translation occurs based on the user's default browser language. Update the Language Settings in the Chrome Extension based on how the translation should occur. If the widget is translating but not the modal, please reach out to Sezzle Support. No other languages are supported at this time.
 
 ## Can the color of the Sezzle logo be changed?
-Yes, you can update the logo to coordinate and contrast with the background color of your website. The Sezzle logo `theme` options available are `dark`, `light`, `grayscale`, `black-flat`, `white`, `white-flat`, `purple-pill`, or `white-pill`. However, it is against our branding guidelines to otherwise customize the logo color.
+Yes, you can update the logo to coordinate and contrast with the background color of your website. The Sezzle logo `theme` options available are `dark`, `light`, `grayscale`, `black-flat`, `white`, `white-flat`, `purple-pill`, `pride-white-pill`,`pride-purple-pill`,`white-pill`. However, it is against our branding guidelines to otherwise customize the logo color.
 
 <div style="background: white; color: #392558; width: fit-content;">
 <span>Light: <img style="height: 18px; padding: 5px; " src='https://media.sezzle.com/branding/2.0/Sezzle_Logo_FullColor.svg' alt=' default color Sezzle logo with dark text'></span>
 <span>Grayscale: <img style="height: 18px; padding: 5px; " src='https://media.sezzle.com/branding/2.0/Sezzle_Logo_Black.svg' alt='black gradient Sezzle logo'></span>
 <span>Black-Flat: <img style="height: 18px; padding: 5px; " src='https://media.sezzle.com/branding/2.0/Sezzle_Logo_BlackAlt.svg' alt='black flat Sezzle logo'></span>
 <span>Purple-Pill: <img style="height: 18px; padding: 5px; " src='https://media.sezzle.com/branding/2.0/Sezzle_Logo_PurplePill.svg' alt='purple pill Sezzle logo'></span>
+<span>Pride-Purple-Pill: <img style="height: 18px; padding: 5px; " src='https://media.sezzle.com/branding/2.0/Sezzle_Logo_Pride_PurplePill.svg' alt='pride purple pill Sezzle logo'></span>
```

Comment:
Same as above - we need to be consistent with our naming of things - use `pride-pill`